src/z3c/celery/tests/test_celery.py

```diff
@@ -256,42 +259,80 @@ def test_celery__TransactionAwareTask__run_in_worker__1__cov(
     """
     eager_celery_app.conf['ZOPE_CONF'] = None
 
-    @z3c.celery.task
-    def unbound_task(context=None, datetime=None):
-        """Task which has not yet been bound.
-
-        This allows to change the configuration of it easily.
-        """
-
-    configure_zope = 'z3c.celery.celery.TransactionAwareTask.configure_zope'
-    with mock.patch(configure_zope):
+    loader = ZopeLoader(app=eager_celery_app)
+    with pytest.raises(ValueError) as err:
+        loader.on_worker_process_init()
+    assert (
+        'Celery setting ZOPE_CONF not set, check celery worker config.' ==
+        str(err.value))
 
-        with pytest.raises(ValueError) as err:
-            unbound_task(_run_asynchronously_=True)
-        assert (
-            'Celery setting ZOPE_CONF not set, check celery worker config.' ==
-            str(err.value))
 
-
-def test_celery__TransactionAwareTask__configure_zope__1__cov(
+def test_celery__ZopeBootstep__1__cov(
         eager_celery_app):
-    """It loads the ZCML file defined in `ZOPE_CONF`."""
-    # There is no example.user defined by default:
-    with pytest.raises(zope.authentication.interfaces.PrincipalLookupError):
-        principalRegistry.getPrincipal('example.user')
```

Comment:
I am missing this assertion in the refactored test. It was there to ensure the base line.